CRM/Civicase/Upgrader.php

```diff
@@ -105,9 +105,9 @@ public function install() {
         'option_group_id' => 'activity_status',
         'name' => $status,
         'return' => 'id',
-        'api.OptionValue.setvalue' => [
-          'field' => 'grouping',
-          'value' => $grouping,
+        'api.OptionValue.create' => [
+          'option_group_id' => 'activity_status',
```

Comment:
So I guess this change follows the new note you left on the PR description which is:

```
In addition, a similar SQL syntax error was reported when installing the extension, this was fixed also.
```

but it is not clear how changing the API call from `setvalue` to `create` fixes that, can you clarify that ?  also anyway for `create` API call to work, you will need to supply the `id` parameter for it to behave like an "update" operation, which is when you used in a chained API call can be done like this:

```
    civicrm_api3('OptionValue', 'get', [
      'option_group_id' => 'activity_status',
      'name' => $status,
      'return' => 'id',
      'api.OptionValue.create' => [
        'id' => '$value.id',
        'grouping' => $grouping,
      ],
    ]);
```





Can you play with these in the API explorer as well as test the extension installation before pushing new updates.

Reply:
Thanks, @omarabuhussein, when chaining API calls, it's really not necessary to specify the id when the id is the only field returned; as seen in the recording below.

![webb](https://user-images.githubusercontent.com/85277674/176898810-a5086788-4482-4f3c-9306-56dad9f931c0.gif)

If you prefer that I should be explicit, let me know.

Reply:
Ah okay, I was not aware of that, can you though explain the PR description how this change fixes the installation issue .